cmds/cluster/storestats/main.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"flag"
+	"fmt"
+	"io"
+	"log"
+	"os"
+	"strings"
+	"time"
+
+	"cloud.google.com/go/storage"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/option"
+)
+
+const (
+	lifecycleConfig = `{
+		"rule": [
+			{
+				"action": {
+					"type": "Delete"
+				},
+				"condition": {
+					"age": 1
+				}
+			}
+		]
+	}`
+)
```

Comment:
What is this? Where does it come from? Where does it go? If unused, may consider removing it :)